pyuvdata/tests/__init__.py

```diff
@@ -26,6 +29,17 @@ def setup_package():
         print('making test directory')
         os.mkdir(testdir)
 
+    # try to download the iers table. If it fails, turn off auto downloading for the tests
+    # and turn it back on in teardown_package (done by extending auto_max_age)
+    try:
+        iers_a = iers.IERS_A.open(iers.IERS_A_URL)
+    except(urllib.error.URLError):
+        iers.conf.auto_max_age = None
+
+
+def teardown_package():
+    iers.conf.auto_max_age = 30
```

Comment:
I know this is the default value, and few people have reason to change it, but it might be nice to save and restore the `iers.conf.auto_max_age` value if we need to reset it to `None`. I think this can be done if we switch to unittest-style `setUpPackage` and `tearDownPackage` methods. Let me know if you'd like me to implement that.

Reply:
You're right, that would be better. If you want to tackle it, go for it!

Reply:
So, it seems that the value of `auto_max_age` is reset between python sessions, so it shouldn't strictly be necessary to reset it after we're done. I don't think it hurts to be thorough, so we should leave it, but there's no danger of it overwriting the user's default.